### PR TITLE
Tune py-3.14 version

### PR DESCRIPTION
**Pull request by asvetlov**
Use the latest available 3.14.0-alpha2

### CODE REVIEW

File: pytest.ini [hunk 1]
```diff
@@ -14,5 +14,6 @@ asyncio_mode = auto
 asyncio_default_fixture_loop_scope = function
 filterwarnings =
     error
+    ignore:'asyncio.iscoroutinefunction' is deprecated:DeprecationWarning:aiohttp
```
Comment (Dreamsorcerer) [inline pytest.ini:17]: Shouldn't we just fix this one?
Looks like it should be changed to inspect.iscoroutinefunction().

Comment (asvetlov) [reply pytest.ini:17]: Yes, we should fix it in aiohttp and temporarily ignore it for aiojobs.
I have a plan to enable py3.14 alpha for aiohttp and see how it is going.

Comment (asvetlov) [reply pytest.ini:17]: Yes, we should fix it in aiohttp and temporarily ignore it for aiojobs.
I have a plan to enable py3.14 alpha for aiohttp and see how it is going.

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -58,7 +58,7 @@ jobs:
          pyver: ['3.9', '3.10', '3.11', '3.12', '3.13']
          experimental: [false]
          include:
-           - pyver: "3.14"
```
Comment (Dreamsorcerer) [inline .github/workflows/ci.yml:61]: This would use the latest release already, so this PR has just stuck us with an older release...

**Pull request merged**